### PR TITLE
replace murmur3 with md5

### PR DESCRIPTION
**Pull request by mthambipillai**
Replace the murmur3 hash function with md5 when converting `partitionKey` to `streamPartition` to reflect the same changes made in the JS and Java clients.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <JSCodeStyleSettings version="0">
```
Comment (kare) [inline .idea/codeStyles/Project.xml]: What version of IDEA you are running? I've got 2018.3.5

File: .idea/misc.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="FLOW" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="1.8.0.171-oracle" project-jdk-type="JavaSDK">
```
Comment (kare) [inline .idea/misc.xml]: This is the production JDK version. I'd like to keep it like that!

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (mthambipillai) [reply .idea/codeStyles/Project.xml]: I have 2018.2.4

Comment (kare) [reply .idea/codeStyles/Project.xml]: Please upgrade!

**Pull request merged**